pkg/apis/application/v1alpha1/types.go

```diff
@@ -1555,7 +1555,7 @@ func validatePolicy(proj string, role string, policy string) error {
 	}
 	// resource
 	resource := strings.Trim(policyComponents[2], " ")
-	if resource != "applications" && resource != "repositories" && resource != "clusters" {
+	if resource != "applications" && resource != "repositories" && resource != "clusters" && resource != "exec" && resource != "logs" {
```

Comment:
While you're here, do you mind cleaning up and copying what we did for validActions:

```golang
var validActions = map[string]bool{
	"get":      true,
	"create":   true,
	"update":   true,
	"delete":   true,
	"sync":     true,
	"override": true,
	"*":        true,
}
```

Reply:
for sure, should I open a new PR or just commit to this one?